lib/main.js

```diff
@@ -353,16 +352,17 @@ export default {
           }
 
           const msg = {
-            type: message.type,
-            filePath,
-            range,
+            severity: message.type.toLowerCase(),
+            location: {
+              file: filePath,
+              position: range,
```

Comment:
Just rename `range` above to position, might as well have it match the new name.